base_layer/core/Cargo.toml

```diff
@@ -63,7 +63,7 @@ once_cell = "1.8.0"
 prost = "0.9"
 prost-types = "0.9"
 rand = "0.7.3"
-randomx-rs = { git = "https://github.com/tari-project/randomx-rs", tag = "v1.1.14", optional = true }
+randomx-rs = { git = "https://github.com/tari-project/randomx-rs", tag = "v1.1.15", optional = true }
```

Comment:
This appears to be a breaking change with this version bump.

The library does no compile now with this error:

```text
error[E0599]: the method `zeroize` exists for mutable reference `&mut <R as rand_core::SeedableRng>::Seed`, but its trait bounds were not satisfied
  --> /Users/cayle/.cargo/git/checkouts/tari-crypto-67e9b858ec4b96fa/66cd1d1/src/deterministic_randomizer.rs:73:10
   |
73 | #[derive(Zeroize)]
   |          ^^^^^^^ method cannot be called on `&mut <R as rand_core::SeedableRng>::Seed` due to unsatisfied trait bounds

```